api/router.go

```diff
@@ -223,11 +223,6 @@ func getRoutes(dt *Dt) []routeHandler {
 			handler: h.deleteBundleHandler,
 			methods: []string{"POST"},
 		},
-		// self test route
-		{
-			url:     baseRoute + "/selftest/info",
```

Comment:
Do you think we should keep this endpoint for backward compatibility even if it returns dummy response?